docs/docs/frigate/installation.md

```diff
@@ -87,12 +87,16 @@ Running in Docker with compose is the recommended install method:
 
 The following officially supported builds are available:
 
-`ghcr.io/blakeblackshear/frigate:stable` - Standard Frigate build for amd64 & RPi Optimized Frigate build for arm64
+`ghcr.io/blakeblackshear/frigate:stable` - Standard Frigate build for amd64
```

Comment:
Why was this changed? This change is incorrect

Reply:
Reverted